src/components/FieldLevelHelp/FieldLevelHelp.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+import { DOCUMENTATION_URL } from '../../../storybook/constants';
+import { FieldLevelHelp } from './index';
+
```

Comment:
maybe we should add it under the forms stories? it makes sense to me to see a complete form with field level help instead of a standalone?

Reply:
In 6c9f1d2, I have added the story under Forms.

It was a good call to add the FieldLevelHelp use case in the Form, since it looks like I may have stumbled upon a bug  in the Popover (or OverlayTrigger) component.

When the Popover component is used in conjunction with the `FormGroup` component, the Formgroup component steals the focus from Popover, causing the popover to close immediately. This happens in a fraction of a second, giving a visual impression that the popover never opened.

Please see a demo of the above issue here - https://aparnakarve.github.io/patternfly-react
under the Forms->Horizontal Form->Phone field

For this particular use case, a workaround would be to keep the Popover open - (An option that you requested here - https://github.com/patternfly/patternfly-react/pull/179#discussion_r163801743). That way, we would at least see the popover contents.
The 'Close Popover' knob demonstrates the above (set the value to false)

The other workaround is to use Tooltips - but since we are not considering tooltips for the FieldLevelHelp component, that is probably not an option.
The 'Popover/Tooltip' knob demonstrates the above (set the value to tooltip)